docs/guide/docs/main/memory.md

```diff
@@ -1,155 +1,105 @@
 ---
 id: memory
-title: Memory
+title: Chatbot Memory- Using Data
 ---
 
-In the course of a conversation, you may want to ask questions to the user, and remember his answers to use it later. You may also want to access the values of system parameters such as the values of the slots that were just extracted.
+In a conversation, you may want to ask questions to the user and remember his answers to use later. You may also want to access the values of system parameters, such as the values of the slots that your chatbot just extracted.
 
 ## System Parameters
 
-When a user engages with a bot, Botpress tracks all the variables and parameters associated with that bot as the bot transitions from one state to another. If you run the emulator you will see the tree of all the system parameters that it tracks.
+When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you will see the tree of all the system parameters that it tracks. Just click the debugger button (circled in red below) and it will open in the bottom panel by default.
 
-You can access these system parameters from the flow builder and from within your code (including in Actions). To do so, all you need to do to reference a parameter by prefixing the path shown in the emulator with “event.”.
+![How to ACcess Debugger](../assets/access-debugger.png)
 
-For example, the path shown in the emulator to the language parameter is nlu.language. You can reference that parameter by adding “event.” to the path shown in the emulator i.e. event.nlu.language.
+You can access these system parameters from the flow builder and within your code (including in Actions). To do so, all you need to do to reference a parameter by prefixing the path shown in the emulator with “event.”.
 
-![NLU Language Emulator](assets/nlu-language-emulator.png)
+For example, the path shown in the debugger to the language parameter is nlu.language. You can reference that parameter by adding “event.” to the path shown in the debugger, i.e. event.nlu.language.
 
-In the Flow Editor you can access system parameters by bracketing them with two sets of curly brackets.
+![NLU Language Emulator](../assets/nlu-emulator.png)
 
-For example in a message you could say:
+In the Flow Editor, you can access system parameters by bracketing them with two sets of curly brackets. For example, in a message, you could say:
 
-The bot's language is {{event.nlu.language}}.
-
-![NLU Language Message](assets/nlu-language-message.png)
+> The chatbot's language is {{event.nlu.language}}.
```

Comment:
I know you didn't write this but it's actually false ... it's not the chatbot's language but the user's input language.

Reply:
Okay, fixed that.